staging/src/k8s.io/api/node/v1alpha1/types.go

```diff
@@ -114,3 +114,69 @@ type RuntimeClassList struct {
 	// Items is a list of schema objects.
 	Items []RuntimeClass `json:"items" protobuf:"bytes,2,rep,name=items"`
 }
+
+// +k8s:conversion-gen:explicit-from=net/url.Values
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// NodeLogViewerOptions is the query options for a Node's logs REST call.
+type NodeLogViewerOptions struct {
+	metav1.TypeMeta `json:",inline"`
+
+	// SinceTime is an RFC3339 timestamp from which to show logs. /If this value precedes the time a log file or service
+	// was created, only logs since the log file or service was created will be returned. If this value is in the
+	// future, no logs will be returned.
+	// +optional
+	SinceTime *metav1.Time `json:"sinceTime,omitempty" protobuf:"bytes,1,opt,name=sinceTime"`
+
+	// UntilTime is an RFC3339 timestamp until which to show logs.
+	// +optional
+	UntilTime *metav1.Time `json:"untilTime,omitempty" protobuf:"bytes,2,opt,name=untilTime"`
+
+	// TailLines if set, return up to this many lines (not more than 100k) from the end of the log. Only applies to
+	// service logs.
+	// +optional
+	TailLines *int64 `json:"tailLines,omitempty" protobuf:"varint,3,opt,name=tailLines"`
+
+	// Path is used to retrieve the specified path within the node's /var/logs/ folder.
```

Comment:
I suggest to change all `/var/logs` references to simply "logs" - "within the node's logs folder".  Leave it vague

Reply:
done